CHANGE_LOG

```diff
@@ -1199,7 +1199,7 @@ Enhancements from user contributed PRs (with thanks!):
 * Lucio Fernandez-Arjona extended Numba's ``np.sum`` support to now accept the
   ``dtype`` kwarg in #4472.
 * Pedro A. Morales Maries added support for ``np.cross`` in #4128 and also added
-  the necessary extension ``numba.numpy_extensions.cross2d`` in #4595.
+  the necessary extension ``numba.np.extensions.cross2d`` in #4595.
```

Comment:
As the change log is a reflection of history I don't think that this should be changed, irrespective of this being in the wrong place for current Numba.

Reply:
Ok my bad, I was doubtful about changing this but thank you for the information